src/lib.rs

```diff
@@ -16,7 +16,7 @@
 
 #![forbid(warnings)]
 #![deny(missing_docs)]
-#![recursion_limit = "1024"]
+#![recursion_limit = "2048"]
```

Comment:
lol, feeling the pain that got you here

Reply:
ja and good catch lets see if that can be reverted now

Reply:
yes it can! Thanks for catching this!